ios/samples/hello-pbr/hello-pbr/CameraManipulator.h

```diff
@@ -17,6 +17,7 @@
 #ifndef TNT_FILAMENT_SAMPLE_CAMERA_MANIPULATOR_H
 #define TNT_FILAMENT_SAMPLE_CAMERA_MANIPULATOR_H
 
+#include <functional>
```

Comment:
Nit: please add an empty line after this include. We keep them grouped by library.